go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/chrismalek/oktasdk-go v0.0.0-20181212195951-3430665dfaa0 // indirect
 	github.com/containerd/continuity v0.0.0-20181203112020-004b46473808 // indirect
 	github.com/coreos/etcd v3.3.11+incompatible // indirect
+	github.com/coreos/go-oidc v2.0.0+incompatible // indirect
```

Comment:
Why is there a new thing added?

Reply:
I only ran `go mod tidy` :-)

Let's see...

```
$ go mod why github.com/coreos/go-oidc
# github.com/coreos/go-oidc
gocloud.dev/secrets/vault
github.com/hashicorp/vault/api
github.com/hashicorp/vault/api.test
github.com/hashicorp/vault/helper/builtinplugins
github.com/hashicorp/vault-plugin-auth-azure
github.com/coreos/go-oidc
```